Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Boxes/magnum.yml

```
@@ -87,6 +91,7 @@
   id: MagazineBoxMagnumAP
   parent: BaseMagazineBoxMagnum
   name: ammunition box (.45 magnum armor-piercing)
+  description: A cardboard box of .45 magnum rounds. Intended to hold restricted armor-piercing ammunition.
```

Restricted yet not under any contraband designation

---

Good point, ill change that

---

Still needs to be changed
```suggestion
  description: A cardboard box of .45 magnum rounds. Intended to hold armor-piercing ammunition.
```

---

how did I miss that?